astropy/utils/console.py

```diff
@@ -815,10 +833,13 @@ def work(i):
                     if (i % chunksize) == 0:
                         bar.update(i)
             else:
-                p = multiprocessing.Pool(
+                p = multiprocessing.get_context(
```

Comment:
Bit nitpicky, but rather than tie these all together, I think it would be clearer to do
```
ctx = multiprocessing.get_context(multiprocessing_start_method)
p = ctx.Pool(process=(...))
```

Reply:
Done.